docs/usage/user.md

```diff
@@ -70,6 +71,18 @@ new_user = client.create_user('Dummy User', 'user@example.com')
 
 [create_user]: https://box-python-sdk.readthedocs.io/en/latest/boxsdk.client.html#boxsdk.client.client.Client.create_user
 
+Get the Avatar for a User
+-------------------------
+
+To get the avatar for a user call the [`user.get_avatar()`][get_avatar] method with the [`User`][user_class] 
+object for the user you wish to retrieve an avatar for.
+
+```python
+avatar = client.user('33333').avatar_url()
```

Comment:
This sample code doesn't call the method you implemented — this should probably be `.get_avatar()`